rust/Cargo.toml

```diff
@@ -7,7 +7,7 @@ authors = ["Giovanni Petrantoni <sinkingsugar@gmail.com>"]
 edition = "2021"
 
 [lib]
-crate-type = ["rlib", "staticlib"]
+crate-type = ["rlib", "staticlib"] 
```

Comment:
Looks like a parasite change: not sure why there should be a whitespace at the end.